packages/admin/cms-admin/src/cronJobs/JobLogs.tsx

```diff
@@ -0,0 +1,93 @@
+import { gql, useQuery } from "@apollo/client";
+import { Loading, MainContent, Toolbar, ToolbarBackButton, ToolbarFillSpace, ToolbarTitleItem } from "@comet/admin";
+import { Box, Typography } from "@mui/material";
+import { styled } from "@mui/material/styles";
+import * as React from "react";
+import { FormattedMessage } from "react-intl";
+
+import {
+    GQLKubernetesJobLogsQuery,
+    GQLKubernetesJobLogsQueryVariables,
+    GQLKubernetesJobQuery,
+    GQLKubernetesJobQueryVariables,
+} from "./JobLogs.generated";
+
+const jobQuery = gql`
+    query KubernetesJob($name: String!) {
+        kubernetesJob(name: $name) {
+            id
+            name
+            label
+        }
+    }
+`;
+
+function JobLogsToolbar(props: { jobName: string }) {
+    const { jobName } = props;
+
+    const { data } = useQuery<GQLKubernetesJobQuery, GQLKubernetesJobQueryVariables>(jobQuery, { variables: { name: jobName } });
```

Comment:
This causes a flicker in the toolbar while the data (= name) is loading, right? Is this noticeable/distracting?

Reply:
Improved with 3e89e0038821b045af1b5fd7912c66528337f36a